locales/en.default.schema.json

```
@@ -1059,6 +1059,20 @@
             "label": "Always, reduce logo size"
           }
         },
+        "transparent_overlay_header": {
+          "content": "Transparent overlay header"
+        },
+        "enable_transparent_header": {
+          "label": "Enable transparent overlay header on homepage"
+        },
+        "transparent_logo": {
+          "label": "Transparent overlay logo",
+          "info": "Only displaed when transparent overlay header is used."
```

```suggestion
          "info": "For best results, use a transparent PNG image. Visible when transparent overlay header is displayed."
```
This aligns more with other language used for similar "conditional" setting".